docs/source/python_api/mlflow.artifacts.rst

```diff
@@ -0,0 +1,7 @@
+mlflow.artifacts
```

Comment:
Document the new module. We opt to introduce `mlflow.artifacts` for parity / consistency with the existing `mlflow artifacts` CLI